sdk/cognitivelanguage/ai-language-text/test/internal/errorTargets.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { createHttpHeaders, PipelineRequest } from "@azure/core-rest-pipeline";
+import { assert } from "@azure/test-utils";
+import { KnownErrorCode } from "../../src/generated";
+import { AnalyzeBatchActionNames } from "../../src/models";
+import { TextAnalysisClient } from "../../src/textAnalysisClient";
+import { extractErrorPointerIndex } from "../../src/util";
+import { expectation73 } from "../public/expectations";
+import { assertActionsResults } from "../public/utils/resultHelper";
+
+describe("Error targets", function () {
+  let client: TextAnalysisClient;
+
+  before(async function () {
+    const response1 = (request: PipelineRequest) => ({
```

Comment:
Let's pull everything in here out into a top-level function that returns the client, similar to this [one](https://github.com/Azure/azure-sdk-for-js/blob/3604079c0c76e932af3c242da73dafe2ede24a81/sdk/cognitivelanguage/ai-language-text/test/internal/warn_text.spec.ts#L10).